packages/ui/src/global.css

```diff
@@ -1,7 +1,17 @@
+@keyframes cl-spin {
+  to {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
```

Comment:
Shared privately; unfortunately we can't use the included `@keyframes` or the `keyframes` key in `tailwind.config.ts`